FrejaEidClient/src/main/java/com/verisec/frejaeid/client/beans/general/SslSettings.java

```diff
@@ -47,16 +50,16 @@ public static SslSettings create(String keystorePath, String keystorePass, Strin
      * <B>JCEKS</B>, <B>PKCS12</B>), password.
      *
      * <B>Use this creator if you have added server certificate to keyStore.</B>
-     *
+     * <p>
```

Comment:
Do we need this?

Reply:
If you're referring to <p>, it was autogenerated by IntelliJ.

Reply:
That is weird. But if it will be always added, then let it stay. :)